aldryn_categories/cms_wizards.py

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.utils.translation import ugettext_lazy as _
+
+from cms.wizards.wizard_pool import wizard_pool
+from cms.wizards.wizard_base import Wizard
+from cms.wizards.forms import BaseFormMixin
+
+from parler.forms import TranslatableModelForm
+from treebeard.forms import movenodeform_factory, MoveNodeForm
+
+from .models import Category
+
+
+class CategoryWizard(Wizard):
+
+    def get_success_url(self, *args, **kwargs):
+        # get category object and if it has page return it's url,
+        # otherwise redirect to root.
+        obj = kwargs['obj']
+        if hasattr(obj, 'page'):
+            return obj.page.get_absolute_url(kwargs['language'])
+        return '/'
```

Comment:
Please review the latest code in get_success_url in the Wizard base. We are forcing users to edit mode now. If you override, then we probably should do the same here.